simple_message/include/simple_message/messages/robot_status_message.h

```diff
@@ -110,8 +110,38 @@ class RobotStatusMessage : public industrial::typed_message::TypedMessage
   {
     return this->status_.byteLength();
   }
+  
+  /**
+   * \brief Gets the communication type of the message
+   * 
+   * \return the value of the comm_type parameter (refer to simple_message::CommTypes::CommType)
+   */
+  int getCommType() const

```

Comment:
This is most likely missing functionality in other req/reply classes as well, so it makes sense to add this to shared parts.